### ISSUE TITLE
Getsparks site not usable

### ISSUE THREAD

**Issue by divineslight** [katzgrau/getsparks.org#12]
I recently had to update spark found out no such option exists,

Installed newest version configured it, and then wanted to remove older one to clean my repository, don't know what option to give to remove a specific version, keep fiddling and it keep saying to give -f, which removed all versions.

The sparks are the awesome thing that happened to CI, please can you create a cool user guide? on getsparks,org?

If you guys put little more effort on making it usable and readying it for the masses it will be a great gift for CI.

Regards,
Munir

**Comment by seejohnrun** [comment]
Hey Minir,

We'll get a more detailed user doc for `sparks` up very soon - I definitely agree that some of the behavior just isn't clear enough in the CLI help

**Comment by divineslight** [comment]
I hope so, also please work on update spark and remove a specific version.

Thanks,

**Comment by seejohnrun** [comment]
For upgrade/install you should be able to run:

`spark install something`

And to remove a specific version:

`spark remove -v1.0.0 something`

Are these not working for you?

**Comment by divineslight** [comment]
Yes installing was not a problem, but like me many will get confused if word "update " is not used.

Also I've been trying

```
remove something -v 1.0.0
```

and

```
remove something 1.0.0
```

because I couldn't find the right syntax anywhere.

Also the error message on wrong syntax for "remove" is pretty confusing 

> Please specify a version of remove all with -f

I guess there's a typo, of should be or.

I will send you a minor pull request....

**Comment by seejohnrun** [comment]
- I'll add an `update` command
- The syntax is a bit confusing, I'll either make it work both ways or add a note
- Yep - should be or
